src/lxc/attach.c

```diff
@@ -339,6 +340,7 @@ static int lxc_attach_set_environment(struct lxc_proc_context_info *init_ctx,
 			for (count = 0; extra_keep[count]; count++)
 				;
 
+			assert(count != 0 && "count is 0");
```

Comment:
We will never ever do `assert()` **anywhere** in the shared library. Also the second conjunct is just weird.